tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -373,6 +373,7 @@ function get_pr_code_friendliness($payload, $oldbalance = null){
 		'Feature' => -1,
 		'Balance/Rebalance' => -1,
 		'Tweak' => -1,
+		'Brain Damage Line' => -10,
```

Comment:
what does this even do

Reply:
#29897

Reply:
https://github.com/tgstation/tgstation/labels/Brain%20Damage%20Line